rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/70_synonym_rule_delete.yml

```diff
@@ -0,0 +1,62 @@
+setup:
+  - skip:
+      version: " - 8.9.99"
+      reason: Introduced in 8.10.0
+  - do:
+      synonyms.put:
+        synonyms_set: test-synonyms
+        body:
+          synonyms_set:
+            - synonyms: "hello, hi"
+              id: "test-id-1"
+            - synonyms: "bye => goodbye"
+              id: "test-id-2"
+            - synonyms: "test => check"
+              id: "test-id-3"
+
+---
+"Delete synonym rule":
```

Comment:
It could be good to add another test case, where we delete a synonym ID that also exists in a separate synonym set, to make sure that only that synonym ID in the synonym set we want to remove gets deleted?

Reply:
I think this is already covered in the overall DELETE operation [here](https://github.com/elastic/elasticsearch/blob/b508ee7886d1237e3ce6dc2d61ed92e3eb2d1091/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/30_synonyms_delete.yml#L39), but it doesn't hurt to add it. Thanks!